### DISCUSSION TITLE
[Projects] Allow Any Repository To Be Added To The Repository Section

### DISCUSSION THREAD

**Discussion by CuriousTommy** [community/community#40599] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

I've been playing around with the new project interface and, for the most part, love the table interface! However, there is one issue I have with it.

---

For the `Repository` section, it seems to only be possible to add in repositories that have their issue tracker enabled. As a workaround, I had to add in a `Repository URL` section to reference repositories that don't have their issue tracker enabled (see picture as an example).

![image](https://user-images.githubusercontent.com/3105794/205214898-633f65f4-5cc5-4298-9798-7ec7171b6d9e.png)

This leaves me with a few options:
* **Option 1:** Stick with the current workaround. Use `Repository` for repos that have their issue tracker enabled and `Repository URL` for repos that don't have it enabled
* **Option 2:** Hide `Repository` and stick with just `Repository URL`
* **Option 3:** Enable the issue tracker for the repo that have it disabled.

I'm not a huge fan of the following options:
* **Option 1** just wastes a bunch of space.
* I wouldn't mind doing **Option 2** if the text field supported markdown. Unfortunately, that does not seem to be the case[1]... I would prefer having the repos be neatly formatted, like they currently are in the `Repository` section.
* For **Option 3**, I might be able to convince the owner of those repos to enable the issue tracker. However, I want to make an argument for why this option may not work for some projects.
  * Some people may not agree with me on this, but I like to see the `Repository` section as a great way to reference the repo for where the code changes will reside in. However, that does not necessarily mean that a ticket should be created in that repo to track the progress. Some projects may only want to have an issue ticket live under one repo.

[1] No markdown support for the text field

![image](https://user-images.githubusercontent.com/3105794/205219129-c6486c5c-4db1-46bf-a422-b52e92c00c9c.png)

---

I think one good option is to allow any type of repo (regardless of whether an issue tracker is enabled or not) to be added to the `Repository` section.

If that is not possible, I think allowing markdown for the text field would be a good alternative.

**Comment by ossanna16** [comment]
Hi there @CuriousTommy and welcome to our community! Thank you for the feedback!
